omnigibson/simulator.py

```diff
@@ -105,8 +105,20 @@ def _launch_app():
         # sys.argv.append("--/log/outputStreamLevel=error")
         warnings.simplefilter("ignore", category=NumbaPerformanceWarning)
 
+    # Try to import the isaacsim module that only shows up when pip installing omniverse
+    try:
+        import isaacsim  # noqa: F401
```

Comment:
interesting -- so when pip installing, are the LD_LIBRARY_PATH / PYTHONPATH env variables auto-updated when importing isaacsim? Formerly we had to sort of a do a hacky thing where every time you activated the conda env to run omnigibson we would automatically run the setup script from isaac so that these paths were updated.

Reply:
Yes, I believe they get updated upon import. Pretty neat.